src/components/Books.jsx

```diff
@@ -1,28 +1,16 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 
 const Books = () => {
-  const books = [
-    {
-      id: 1,
-      title: 'Romeo and Julliet',
-      author: 'William Shakespare',
-      category: 'Romance',
-    },
-    {
-      id: 2,
-      title: 'The Hunger Games',
-      author: 'Suzanne Collins',
-      category: 'Action',
-    },
-  ];
+  const { books } = useSelector((state) => state.book);
+  console.log(books);
 
   return (
-    <ul>
-      {books.map((book) => (
-        <Book key={book.title} title={book.title} author={book.author} category={book.category} />
-      ))}
-    </ul>
+    <div>
+      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
```

Comment:
- [ ] I highly recommend not disabling the Linters since you should solve the issue not disable it because the Linters help you to write clean code and dry 🤩🔥